scripts/release.sh

```diff
@@ -38,6 +38,24 @@ mkdir -p dist/
 echo "> Checkout version branch"
 git checkout -B "${git_branch}"
 
+# Please update this part while package list changed.
+echo "> Update Cargo.toml to remove not released members"
+PACKAGES=(
```

Comment:
Missing Zig Swift and go. May we sort this?

Reply:
They are not listed inside `Cargo.toml`.

Reply:
Confirmed.